x-pack/filebeat/module/aws/elb/README.md

```diff
@@ -0,0 +1,64 @@
+Filebeat module for AWS ELB
+===
+
+Module for the AWS load balancers, it supports the following flavours:
+
+* ELB (Classic Load Balancer)
+* Application Load Balancer (V2 Load Balancer for HTTP)
+* Network Load Balancer (V2 Load Balancer for TCP and UDP - UDP not tested)
+
+Implementation based on the description of the access logs from the
+documentation that can be found in:
+
+* ELB: https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/access-log-collection.html
+* Application LB: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
+* Network LB: https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-access-logs.html
+
+Test files starting with `example` are copied or based on examples of this
+documentation.
+
+
+How to manual test this module
+===
+
+* Create an ELB and enable access logs for it. A terraform scenario is included
+  as example, read the section below about this if you want to use it.
+* Make some requests to the load balancer.
+* Configure filebeat, using the queue url from `terraform output sqs_queue_url`.
+```
+filebeat.modules:
```

Comment:
Hmm do we need to add this part into `aws/_meta/config.yml`?

Reply:
Yes, good catch.

Reply:
Added.